docs/user-guide/howto.rst

```diff
@@ -386,13 +385,3 @@ using a dummy phase column.
 
 .. accordion-footer::
 
-.. accordion-header::
```

Comment:
why removing this?

Reply:
Because it was a duplicate -- see line 43